software/control/src/QPReactiveRecoveryPlan.cpp

```diff
@@ -847,9 +855,9 @@ std::map<SupportLogicType, std::vector<bool>> createSupportLogicMaps() {
 
 void QPReactiveRecoveryPlan::encodeSupportData(const int body_id, const FootState &foot_state, const SupportLogicType &support_logic, drake::lcmt_support_data &support_data) {
   support_data.timestamp = 0;
-  support_data.body_id = body_id + 1;
+  support_data.body_name = this->robot->getBodyOrFrameName(body_id);
   support_data.contact_pts.resize(3);
-  Matrix3Xd all_contacts = this->heelToeContacts(body_id);
+  Matrix3Xd all_contacts = this->robot->bodies[body_id]->contact_pts;
```

Comment:
is this ok? It appears to work in practice, but this means the recovery planner is now thinking about all 8 contact points on the feet, not just the heel and toe.